includes/MediaWiki/ReconciliationService.php

```diff
@@ -105,7 +106,9 @@ public function getOrCreateItemByStatementUrl(
 
 		// If we have more than one item matches, something is wrong and we can't edit
 		if ( count( $itemsThatReferenceTheUrlInCorrectStatement ) > 1 ) {
-			die( 'Matched multiple Items during reconciliation :(' );
+			throw new ReconciliationException(
+				MessageValue::new( 'wikibasereconcileedit-reconciliationservice-matched-multiple-items' )
```

Comment:
The message arguments (items, number of items) seems to be missing here.

Reply:
True! :scream: